components/xsd-fu/templates-cpp/DummyMetadata.template

```diff
@@ -466,54 +480,28 @@ namespace ome
 {% end header %}\
 {% if fu.SOURCE_TYPE == "source" %}\
       const std::string&
-      AggregateMetadata::get${abstractClass}Type(${indexes_string(v)}) const;
+      DummyMetadata::get${abstractClass}Type(${indexes_string_dummy(v)}) const
       {
-        throw MetadataException("AggregateMetadata", "get${abstractClass}Type",
+        throw MetadataException("DummyMetadata", "get${abstractClass}Type",
                                 "intentionally not implemented");
       }
 {% end source %}\
 
 {% if fu.SOURCE_TYPE == "header" %}\
         // Documented in base class.
         index_type
-        get${abstractClass}Count(${indexes_string(v)}) const;
+        get${abstractClass}Count(${indexes_string(v[:-1])}) const;
```

Comment:
Never for headers, only for sources where we don't use the arguments.  The sole reason for doing this is to avoid compiler warnings about unused arguments.  They all look consistent from that point of view.